### PR TITLE
Add ELIXIR OIDC backend, upgrade social-auth-core requirement version

### PR DESCRIPTION
**Pull request by kidanem**

### CODE REVIEW

**Comment by bgruening** [comment]
Thanks @kidanem!

Documentation by @kjellp can be found here https://github.com/galaxyproject/galaxy-hub/pull/518

File: config/oidc_backends_config.xml.sample [hunk 1]
```diff
@@ -100,4 +100,13 @@ Please mind `http` and `https`.
         <!-- (Optional) Override the default idp hint -->
         <!-- <idphint>cilogon</idphint> -->
     </provider>
+    
+    <! -- Documentation: https://docs.google.com/document/d/1vOyW4dLVozy7oQvINYxHheVaLvwNsvvghbiKTLg7RbY-->
```
Comment (bgruening) [inline config/oidc_backends_config.xml.sample]: This link should probably go to https://galaxyproject.org/authnz/config/oidc/idps/elixir-aai/ assuming this is merged.

Comment (kjellp) [reply config/oidc_backends_config.xml.sample]: Yes, the link should be updated as you suggest @bgruening . @kidanem could you update the doc link in the backends sample file? The Elixir AAI google link is now available within the galaxyproject.org docs.

Comment (kidanem) [reply config/oidc_backends_config.xml.sample]: @kjellp - the link above doesn't work. Please provide a working link and I will update

**Comment by kjellp** [comment]
It will only work when the PR for the docs is accepted😊 So it is the
correct one😎

tir. 30. apr. 2019, 10:11 skrev Kidane M. Tekle <notifications@github.com>:

File: config/oidc_backends_config.xml.sample (hunk 1 above)
Comment (kidanem) [reply config/oidc_backends_config.xml.sample]: Link updated!

**Comment by nsoranzo** [comment]
@galaxybot test this

**Review by VJalili — APPROVED** [review]

**Comment by VJalili** [comment]
LGTM, thank you @kidanem. +1

**Pull request merged**